packages/telemed-ehr/app/src/telemed/features/appointment/AppointmentSidePanel.tsx

```diff
@@ -179,9 +179,11 @@ export const AppointmentSidePanel: FC<AppointmentSidePanelProps> = ({ appointmen
             )}
           </Box>
 
-          {appointmentType === 'in-person' && (
+          {appointmentType === 'in-person' && isPractitionerAllowedToCancelThisVisit && (
```

Comment:
why?

Reply:
Because the point of `isPractitionerAllowedToCancelThisVisit` variable is regulating provider permission, which depends on the practitioner license and if the patient is assigned to that provider.

Reply:
They should be allowed to cancel but not assign themselves no?

Reply:
I don't think they should be allowed to cancel an appointment they are not assigned to. There is already a logic for this in the code, just commented out because of the licensing issues back then, you can check it out

Reply:
ok, please open a ticket